src/Knp/Menu/Util/MenuManipulator.php

```diff
@@ -197,9 +206,10 @@ public function toArray(ItemInterface $item, ?int $depth = null): array
      *   * ['subItem1', 'subItem2']
      *   * [['label' => 'subItem1', 'url' => '@homepage'], ['label' => 'subItem2']]
      *
-     * @param mixed $subItem A string or array to append onto the end of the array
+     * @param string|ItemInterface|array<int|string, mixed>|\Traversable<mixed> $subItem A string or array to append onto the end of the array
```

Comment:
We could add a more precise type for the array and iterator instead of `mixed`:

```
@phpstan-param string|ItemInterface|array<int|string, string|int|float|null|array{label: string, url: string|null, item: ItemInterface|null}|ItemInterface>|\Traversable<string|int|float|null|array{label: string, url: string|null, item: ItemInterface|null}|ItemInterface> $subItem
```

Reply:
and then, this should also be applied to callers (the templating helper and the twig extension).
This should remove the need for the baseline entry on that method.

Reply:
Well, it looks like is not enough. PHPStan is still complaining